src/components/CallbackForm/CallbackForm.js

```diff
@@ -220,16 +222,19 @@ export default function CallbackForm({residentId, resident, helpRequest, backHre
         // jeez, this looks sooo fragile
         if (!validateCEVNeedsFieldset()) {
             setErrorsExist(true);
+            window.scrollTo(0, 0);
         }
         else if (
-            (callMade == true && callOutcomeValues.length > 1 && callDirection != null && helpNeeded != null && ((email != null && showEmail) || !showEmail) && ((phoneNumber != null && showText) || !showText)) ||
+            isNonEmpty(helpNeeded) && isNonEmpty(callMade) && isNonEmpty(followUpRequired) &&
```

Comment:
Would it behave differently if it was `helpNeeded && callMade && followUpRequired` instead of `isNonEmpty(helpNeeded) && isNonEmpty(callMade) && isNonEmpty(followUpRequired)` ?

Reply:
Yes, it would.

I'm attempting to check whether the inputs were selected by a user. So in essence, I'm checking whether they have been given a value, or not.

The difference between approaches is that the `isNonEmpty` function allows for `false` boolean value. In effect, the validation should not fail because a user has selected "Call Made?" as "No."

If I were to do `helpNeeded && callMade && followUpRequired`, then if a user has selected a "Call Made?" as "No.", then the expression is substituted to `helpNeeded && false && followUpRequired`, which evaluates to `false`.